js/main.js

```diff
@@ -19,6 +19,86 @@ function submission(event) {
   formObject.entryId = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(formObject);
-  $image.src = 'images/placeholder-image-square.jpg';
+  $newImage.src = 'images/placeholder-image-square.jpg';
   $form.reset();
+  renderEntry(data.entries[0]);
+  toggleNoEntries();
+  viewEntries();
+}
+
+function renderEntry(entry) {
+  var $ul = document.querySelector('ul');
+  var $li = document.createElement('li');
+  $li.setAttribute('class', 'row');
+  $ul.appendChild($li);
+
+  var $divImg = document.createElement('div');
+  $divImg.setAttribute('class', 'column-half');
+  $li.appendChild($divImg);
+
+  var $img = document.createElement('img');
+  $img.src = entry.imageUrl;
+  var imgEntry = 'img' + entry.entryId;
+  $img.setAttribute('id', imgEntry);
+  $divImg.appendChild($img);
+
+  var $divText = document.createElement('div');
+  $divText.setAttribute('class', 'column-half');
+  $li.appendChild($divText);
+
+  var $h2 = document.createElement('h2');
+  $h2.textContent = entry.title;
+  var h2Entry = 'h2' + entry.entryId;
+  $h2.setAttribute('id', h2Entry);
+  $divText.appendChild($h2);
+
+  var $p = document.createElement('p');
+  $p.textContent = entry.notes;
+  var pEntry = 'p' + entry.entryId;
+  $p.setAttribute('id', pEntry);
+  $divText.appendChild($p);
+  return $ul;
+}
+
+document.addEventListener('DOMContentLoaded', getEntry);
+
+function getEntry(entry) {
+  for (var i = 0; i < data.entries.length; i++) {
+    renderEntry(data.entries[i]);
+    toggleNoEntries();
+    viewEntries();
```

Comment:
these two functions only need to be called once, not on every iteration of the loop. This is a lot of wasted effort.